lib/telemetry_metrics/console_reporter.ex

```diff
@@ -97,7 +97,12 @@ defmodule Telemetry.Metrics.ConsoleReporter do
               cond do
                 is_nil(measurement) ->
                   """
-                  No value available (metric skipped)
+                  Measurement value missing (metric skipped - handler may detach!)
```

Comment:
It is documented that handlers should automatically skip missing measurements. If the handler detaches it is a bug that must be fixed, it is not a user concern.